testing/database/neo4j-3.2/build.gradle

```diff
@@ -0,0 +1,17 @@
+plugins {

```

Comment:
There is also 3.3.0 now. Can we drop it here as well?

Reply:
done